src/api/projects-iguazio-api.js

```diff
@@ -14,6 +14,20 @@ export default {
       }
     })
   },
+  getProjectMembersVisibility: project => {
+    return iguazioHttpClient.get(`/projects/__name__/${project}`, {
+      params: {
+        include: 'authorization?action=update&sub_resource=authorization/roles'
+      }
+    })
+  },
+  getProjectOwnerVisibility: project => {
+    return iguazioHttpClient.get(`/projects/__name__/${project}`, {
+      params: {
+        include: 'authorization?action=update&sub_resource=authorization/owner'
```

Comment:
URLs are wrong.
`authorization` should bar part of the path.
There should not be an `include` URL param. There should be `action` and `sub_resource` params.

```suggestion
    return iguazioHttpClient.get(`/projects/__name__/${project}/authorization`, {
      params: {
        action: 'authorization/roles',
        sub_resource: 'authorization/roles'
      }
    })
  },
  getProjectOwnerVisibility: project => {
    return iguazioHttpClient.get(`/projects/__name__/${project}/authorization`, {
      params: {
        action: 'update',
        sub_resource: 'authorization/owner'
```